Sources/tuist/TuistApp.swift

```diff
@@ -9,10 +9,22 @@ import TuistSupport
 @_documentation(visibility: private)
 private enum TuistServer {
     static func main() async throws {
+        if CommandLine.arguments.contains("--quiet") && CommandLine.arguments.contains("--verbose") {
+            throw TuistAppError.exclusiveOptionError("quiet", "verbose")
+        }
+
+        if CommandLine.arguments.contains("--quiet") && CommandLine.arguments.contains("--json") {
+            throw TuistAppError.exclusiveOptionError("quiet", "json")
+        }
```

Comment:
What do you think about defining an array of exclusive options and test for any possible combination? `--json` and `--verbose` should be also exclusive.